### PR TITLE
STELLAR-1193 Flip waterfall tick marks

### PR DESCRIPTION
**Pull request by reiinakano**
Follow up to previous PR. I needed to flip the y-axis tick marks as well.

### CODE REVIEW

File: gr-starcoder/python/plot_waterfall.py [hunk 1]
```diff
@@ -101,8 +101,8 @@ def plot_waterfall(arr, samp_rate, center_freq, rps, fft, filename):
         ax.set_yticklabels(y_tick_labels, size='x-large')
     ax.grid(b='on', linestyle='dashed', linewidth=1,
             color='#000000', alpha=0.3)
-    minor_locator_y = IndexLocator(10 * rps, len(arr) % (10 * rps))
-    ax.get_yaxis().set_minor_locator(minor_locator_y)
+    ax.get_yaxis().set_minor_locator(IndexLocator(10 * rps, 0))
+    ax.get_yaxis().set_major_locator(IndexLocator(60 * rps, 0))
```
Comment (gavinreaney) [inline gr-starcoder/python/plot_waterfall.py]: Do you need to explicitly set the major locator now even although it wasn't set before?

Comment (reiinakano) [reply gr-starcoder/python/plot_waterfall.py]: Yeah. I actually think it wasnt properly aligned before either. It wasnt noticeable at all before though, something about having 0 at the bottom made it stand out a bit

**Comment by reiinakano** [comment]
> When we changed the origin to be bottom left, did that cause the tick labels to flip their origin to top-right? I guess so and on that basis the change looks good.

No, the tick labels were top to bottom before, so it was actually aligned with the waterfall. When I flipped the image, the ticks didn't follow it. Now the issue should be fixed.

**Comment by gavinreaney** [comment]
Are you certain? Previously Andrew sent me this link which has tick marks from bottom to top (well before your fix):
https://console.cloud.google.com/storage/browser/telemetry-data-dev/satellites/1/antennas/9/507904/WATERFALL?project=infostellar-cluster

That was what prompted me to ask for the data orientation to be checked because it seemed to not match.

**Comment by reiinakano** [comment]
Oh, you're absolutely right.

> When we changed the origin to be bottom left, did that cause the tick labels to flip their origin to top-right? I guess so and on that basis the change looks good.

is correct then. My bad! :)

**Pull request merged**